packages/seal/src/session-key.ts

```diff
@@ -177,12 +203,12 @@ export class SessionKey {
 	 * Restore a SessionKey instance for the given object.
 	 * @returns A new SessionKey instance with restored state
 	 */
-	static import(
+	static async import(
```

Comment:
this one does not need to check the package again

Reply:
both checks are sanity in case of user error, and worse case it will be detected when calling the server.